frontend/src/components/ItemMetadataForm.js

```diff
@@ -35,13 +56,93 @@ const ItemMetadataForm = ({
     onChange={ event => onChange && onChange({ description: event.target.value }) }
     rows={ 4 }
     value={ description }
+    InputLabelProps={ { shrink: true } }
+  />
+
+  <FormControlLabel
+    checked={downloadable}
+    onChange={ event => onChange && onChange({ downloadable: event.target.checked }) }
+    control={
+      <Checkbox
+      />
+    }
+    label="Allow media to be downloaded and indexed by search engines"
+  />
+
+  <Grid container spacing={8}>
+    <Grid item xs={12} sm={6}>
+      <TextField
+        className={classes.publishedAt}
+        fullWidth
+        error={ !!errors.publishedAt }
+        helperText={ errors.publishedAt ? errors.publishedAt.join(' ') : null }
+        defaultValue={publishedAt ? moment(publishedAt).format(DATETIME_LOCAL_FORMAT) : ''}
+        label="When the item will be published"
+        type="datetime-local"
```

Comment:
Investigate something like [datetime-local-polyfill](https://github.com/jonstipe/datetime-local-polyfill) for IE11.